### PR TITLE
Add functionality to annotate schema files

### PR DESCRIPTION
**Pull request by pascalfleury**
This adds the functionality to annotate all Types and Properties of a file with `isPartOf`.

This also includes a refactoring of the graph logic into a `SchemaOrgGraph` class, which should be then reusable across several parts of the code base.

### CODE REVIEW

File: software/util/reorg.py [hunk 1]
```diff
@@ -6,67 +6,106 @@
...
 
-def SchemaOrgGraph():
-  """Creates a graph with usual namespaces set as we want in schemaorg."""
-  g = Graph()
-  g.bind("dc", Namespace("http://purl.org/dc/terms/"), replace=True)
-  return g
+SCHEMAORG = rdflib.Namespace("https://schema.org/")
+
+class SchemaOrgGraph(object):
+  def __init__(self, filename: str = None, format: str = "turtle"):
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:18]: Nitpick: 4 tab indentation https://google.github.io/styleguide/pyguide.html

File: software/util/reorg.py [hunk 2]
```diff
@@ -6,67 +6,106 @@
...
+    if filename:
+      self.g.parse(filename, format=format)
+
+  def __getattr__(self, *args, **kwargs):
+    return getattr(self.g, *args, **kwargs);
+
+  def IdenticalTo(self, other: "SchemaOrgGraph"):
+    only_in_other = other.g - self.g
+    only_in_self = self.g - other.g
+    if len(only_in_other) or len(only_in_self):
```
Comment (MatthiasWiesmann) [inline software/util/reorg.py:33]: Nitpick, use implicit false, 

if only_in_other or  only_in_self:
  …


see https://google.github.io/styleguide/pyguide.html

Comment (pascalfleury) [reply software/util/reorg.py]: done.

File: software/util/reorg.py (hunk 1 above)
Comment (pascalfleury) [reply software/util/reorg.py]: done

**Review by MatthiasWiesmann — APPROVED** [review]

**Pull request merged**